docs/source/getting_started/upload_dataset.rst

```diff
@@ -0,0 +1,80 @@
+..
+ Copyright 2022 Graviti. Licensed under MIT License.
+
+################
+ Upload Dataset
+################
+
+*************************
+ Create or Get a Dataset
+*************************
+
+.. code:: python
+
+   from graviti import DataFrame, Workspace
+   import graviti.portex as pt
+
+   ws = Workspace(f"{YOUR_ACCESSKEY}")
+   dataset = ws.datasets.create("Graviti-dataset-demo")
+
+***********************
+ DataFrame Preparation
+***********************
+
+Users need to organize the data into :doc:`/features/sheet_management/dataframe` format with
+:doc:`/features/sheet_management/schema`. In addition, SDK also supports uploading different
+kinds of :doc:`/features/sheet_management/binary_files`, such as image, audio, etc.
+
+.. code:: python
+
+   std = pt.build_package("https://github.com/Project-OpenBytes/portex-standard", "main")
+   box2ds = std.label.Box2D(
+       categories=["boat", "car"],
+       attributes={
+           "difficult": pt.boolean(),
+           "occluded": pt.boolean(),
+       },
+   )
+   schema = pt.record(
+       {
+           "filename": pt.string(),
+           "box2ds": pt.array(box2ds),
+       }
+   )
+
+   filenames = ["a.jpg", "b.jpg", "c.jpg"]
+   data = []
+   for filename in filenames:
+       row_data = {
+           "filename": filename,
+           "box2ds": [
+               {
+                   "xmin": 1,
+                   "ymin": 1,
+                   "xmax": 4,
+                   "ymax": 5,
+                   "category": "boat",
+                   "attribute": {
+                       "difficult": False,
+                       "occluded": False,
+                   },
+               },
+           ],
+       }
+       data.append(row_data)
+   dataframe = DataFrame(data=data, schema=schema)
+
+******************
+ Upload DataFrame
+******************
+
+SDK provides a simplified way to upload and modify data:
```

Comment:
I think is not simplified way, it should be normal way, and using draft is complex way.